jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+
+After your build is running successfully with build processing enabled `.circleci/config.yml` file version 2.1, it is possible to add new keys to bundle config stanzas and run the same job more than once with different parameters (re-use jobs).
+
+## Using the `commands` Key and Authoring commands
+
+**Note:** The `commands` stanza is available in configuration version 2.1 and later.
+
+Commands are reusable sets of steps that can be invoked with specific parameters inside a job. For instance, `checkout` and `run` are considered built-in commands. You may also author your own commands or use those authored by others.
+
+### Using Commands
+{:.no_toc}
+
+Commands can be invoked as steps in a job. For instance, to invoke the command `sayhello`, passing it a parameter `to` you would write:
+
+```yaml
+jobs:
+  myjob:
+    docker:
+      - image: "circleci/node:9.6.1"
+    steps:
+      - sayhello:
+          to: "Lev"
+```
+
+## Authoring Commands
+A command definition is YAML that defines metadata, parameters, and a sequence of steps to be executed when invoking the command.
+
+### Command Syntax
+{:.no_toc}
+
+A command can have the following immediate children keys as a map:
+
+- **description:** (optional) A string that describes the purpose of the command, used for generating documentation.
+- **parameters:** (optional) A map of parameter keys, each of which adheres to the `parameter` spec.
+- **steps:** (required) A sequence of steps run inside the calling job of the command.
+
+### Defining Your Commands
+{:.no_toc}
+
+Commands are declared under the `commands` key of a `config.yml` or `orb.yml` file. For instance, to make a command called `sayhello` it might look like:
+
+```yaml
+commands:
+  sayhello:
+    description: "A very simple command for demonstration purposes"
+    parameters:
+      to:
+        type: string
+        default: "Hello World"
+    steps:
+      - run: echo << parameters.to >>
+```
+
+When invoking a command, the steps of that command are inserted where it's invoked. Commands can only be invoked as part of the sequence under `steps` in a job.
+
+### Invoking Other Commands in Your Command
+{:.no_toc}
+
+Commands can use other commands in the scope of execution. 
+
+<!--
+For instance, if a command is declared inside your Orb it can use other commands in that orb. It can also use commands defined in other orbs that you have imported (for example `some-orb/some-command`).
+-->
+
+## Built-in Commands
+
+CircleCI has several built-in commands available to all [circleci.com](http://circleci.com) customers and available by default in CircleCI server installations. Examples of built-in commands are:
+
+  * `checkout`
+  * `setup_remote_docker`
+  * `save_to_workspace`
+
+**Note:** Built-in commands are implicitly in the empty scope and are thus syntactically equivalent to primitives such as `run`). This _may_ change in future versions of configuration but is true to maintain compatibility with version `2` configuration.
```

Comment:
I think we can remove this -- it's not quite true. However, it IS possible to override the built-in commands, so we should probably say that here instead.

Reply:
thanks, removed and replaced with 'it is possible to override built-in commands with a custom command.